Makefile

```diff
@@ -15,44 +16,40 @@ goto_src=cd ${src_dir} && source ${venv_dir}/bin/activate
 help:
 	@echo "Make tasks for installation and deployment. Refer to the Makefile content to know more."
 
-logs:
-	@${remote} "tail -f ~/log/preprod.log"
+logs:  # type=error|access
+	@${remote} "tail -f ~/log/${type}.log"
 
-flush-logs:
-	@${remote} "echo '' > ~/log/preprod.log"
+flush-logs:  # type=error|access
+	@${remote} "echo '' > ~/log/${type}.log"
 	@echo "Log file emptied!"
 
-deploy:
-	@echo "> Fetching master branch and updating sources."
+deploy:  # env=prod|preprod
+	@echo "\n> Fetching master branch and updating sources."
 	${remote} "${goto_src} && git fetch origin ${branch} && git checkout ${branch} && git reset --hard FETCH_HEAD"
 	${remote} "${goto_src} && pip install -r requirements.txt"
-	make rsync
-	@echo "> Launching gunicorn daemon."
-	${remote} "${goto_src} && pkill gunicorn; \
-		gunicorn --daemon -b 127.0.0.1:5000 preprod:app \
-		--error-logfile ~/log/preprod.log --access-logfile ~/log/preprod.log"
-	@echo "> App is deployed. Run \`make logs\` to follow activity."
-
-rsync:
-	@echo "> Synchronizing archives media from old production server."
+	make sync_archives_media
```

Comment:
I tried to keep consistent with dashes in command name, bad idea?

Reply:
Good to me. I was waiting for your other PR to be merged to minimize conflicts in name.